content/applications/finance/payment_providers.rst

```diff
@@ -260,6 +273,15 @@ and by ticking the :guilabel:`Allow Express Checkout` checkbox.
 .. note::
    All prices shown in the express checkout payment form are always taxes included.
 
+.. _payment_providers/features/payment_fees:
+
+Extra payment fees
+------------------
+
+Some payment providers (:doc:`PayPal <payment_providers/paypal>` and :doc:`Alipay
+<payment_providers/alipay>`) support charging payment fees to the customer. If available, you can
+activate this feature by thicking *Add Extra Fees* in the *Fees* tab when configuring the provider.
```

Comment:
@toaa-odoo Also wrote [something about extra fees in his PR](https://github.com/odoo/documentation/pull/2736/files#diff-3e7de8e407a548ca3ef8a8ee8b09dc62653e846b671a75f3100d0180d97825f9R270-R284). Could you adapt your diff with his description of extra fees?

Reply:
Ok, thanks for the tip. It's done. 
@toaa-odoo  I did minor modifications (e.g. both variable and fixed can be activated together). You'll see that when you rebase (if my commit goes first). Also, pay attention to the table above on payment providers: I added a column for extra fees. Please thick it for Demo as well.